pyproject.toml

```diff
@@ -53,6 +53,9 @@ testing = [
     "types-Jinja2 == 2.11.9"
 ]
 
+[project.entry-points."qiskit.transpiler.translation"]
+iqm_n-star = "iqm.qiskit_iqm.iqm_qiskit_plugin:TranslateIQMPlugin"
```

Comment:
Where is this used? Should at least be in the Readme,

Reply:
We decided to drop the Qiskit plugin from the public package, as there is no proper integration to Qiskit transpilation flow. See PR description from details.